homeassistant/components/fujitsu_hvac/coordinator.py

```diff
@@ -57,7 +55,8 @@ async def _async_update_data(self) -> dict[str, FujitsuHVAC]:
             )
 
         try:
-            await gather(*[dev.async_update() for dev in devices])
+            for dev in devices:
+                await dev.async_update()
```

Comment:
The nice thing about gather is that it runs all updates in parallel. This changes it to sequential.

Reply:
Yes, but the tests fail whenever I put it back to the old ones, soooooo